org.jacoco.core.test/src-java7/org/jacoco/core/test/filter/TryWithResourcesTest.java

```diff
@@ -44,7 +44,7 @@ public void test() {
 		// without filter next line has branches:
 		assertLine("test.close", ICounter.EMPTY);
 		assertLine("test.catch", ICounter.NOT_COVERED);
-		assertLine("test.finally", ICounter.PARTLY_COVERED);
+		assertLine("test.finally", ICounter.FULLY_COVERED);
```

Comment:
@Godin Just out of curiosity: The new FinallyFilter now also applies to tryWithResouces?

Reply:
@marchof `javac` translates `try-with-resources` into `try-catch-finally`, so for it for sure applies. However most likely not applies for bytecode that `ecj` generates for closing of resources in `try-with-resources`. But applies for `try-with-resources` with `finally` for both `ecj` and `javac` as this test shows. Can add more precise tests about this, but definitely would prefer to do this in a separate pull request.

Reply:
@Godin Ok, makes sense. I was wondering whether our validadtion tests for filters should only use the filter under test.

Reply:
@marchof we already discussed this: for start it is very good to see combined behavior - this way with low investment we observe what end-user will observe and we observe that filters correctly cooperate and do not interfere with each other in a bad way. However I agree that in long run ability to execute filters separately will be also valuable.

Reply:
@Godin Sure. For I don't want to change test strategy.